torch/csrc/autograd/profiler.cpp

```diff
@@ -114,7 +115,7 @@ void pushRange(std::string name) {
   pushRangeImpl(StringView(std::move(name)));
 }
 
-void popRange() {
```

Comment:
This isn't related though it makes debugging a lot easier when one is logging the `Events` in the cpp code (because it records the name of the op in the event). For example I had to log them to determine why they don't match up when investigating this bug. Although, it's not strictly necessary and I can revert these changes if you prefer that.

Reply:
For future PR, that would be good to have them as a different PR in the stack. You can leave it here for this one, no problem.

Reply:
could you please split this into a separate PR? I think it doesn't belong in this PR and it should be very easy to do